inference-engine/ie_bridges/c/samples/hello_nv12_input_classification/main.c

```diff
@@ -81,7 +111,14 @@ size_t read_image_from_file(const char *img_path, unsigned char *img_data, size_
     return read_size;
 }
 
-size_t parse_image_size(const char *size_str, size_t *width, size_t *height) {
+/**
+* @brief Get image width and height
+* @param string image size in WIDTHxHEIGHT format
+* @param pointer to image width
+* @param pointer to image height
+* @return int status 0(success) or -1(fail)
+*/
+int parse_image_size(const char *size_str, size_t *width, size_t *height) {
```

Comment:
short int? But I think bool is more suitable here

Reply:
updated to bool